src/ignoreFileHandler.ts

```diff
@@ -84,6 +85,10 @@ function getIgnorePathForFile(
     if (!ignorePath) {
         return null;
     }
+    if (!existsSync(ignorePath)) {
+        addToOutput("Wrong prettier.ignorePath provided in your User or Workspace Settings. The path does not exist.");
```

Comment:
It also could be from Folder settings. Shorten to settings.
We should also print `ignorePath` saying it doesn't exist.

Reply:
Doing it ASAP.

Reply:
Updated, please check.